tests/testthat/test-data-combined.R

```diff
@@ -0,0 +1,1072 @@
+# data needed for tests ----------------------------
+
+# will be used across all tests in this file
+
+# load the simulation
+sim <- loadTestSimulation("MinimalModel")
+simResults <- importResultsFromCSV(
+  simulation = sim,
+  filePaths = getTestDataFilePath("Stevens_2012_placebo_indiv_results.csv")
+)
+
+# import observed data (will return a list of DataSet objects)
+dataSet <- loadDataSetsFromExcel(
+  xlsFilePath = getTestDataFilePath("CompiledDataSetStevens2012.xlsx"),
+  importerConfiguration = DataImporterConfiguration$new(getTestDataFilePath("ImporterConfiguration.xml"))
+)
+
+# same as dataSet, but with deliberately changed entries for testing
+dataSet2 <- loadDataSetsFromExcel(
```

Comment:
Instead of creating a new file, you can modify `DataSet` objects by calling `setValues`.